src/renderer/screens/manager/AppsList/UpdateAllApps.js

```diff
@@ -60,6 +67,11 @@ const UpdateAllApps = ({ state, dispatch, isIncomplete, progress }: Props) => {
 
   const { apps, installed, installQueue, uninstallQueue } = state;
 
+  const outOfMemory = useMemo(
+    () => isOutOfMemoryState(predictOptimisticState(reducer(state, { type: "updateAll" }))),
+    [state],
+  );
```

Comment:
perf to be checked. (memo is almost always recalc because state often changes. might offer better efficient ways on live-common side)